### PR TITLE
dbuild: Support build test with dbuild

### PR DESCRIPTION
**Pull request by jeongarmy**
Support build test with dbuild to allow users to check build errors before pushing commits

### CODE REVIEW

File: os/dbuild.sh [hunk 1]
```diff
@@ -154,6 +157,14 @@ function SELECT_OPTION()
 	fi
 }
 
+function BUILD_TEST()
+{
+	echo "Build Test START!! Docker Image Version : ${DOCKER_VERSION}"
```
Comment (sunghan-chang) [inline os/dbuild.sh:162]: Why is docker image version necessary?

File: os/dbuild.sh [hunk 2]
```diff
@@ -154,6 +157,14 @@ function SELECT_OPTION()
...
+	pushd ${OSDIR} > /dev/null
+	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged tizenrt/tizenrt:${DOCKER_VERSION} bash -c "./tools/build_test.sh"
```
Comment (sunghan-chang) [inline os/dbuild.sh:164]: If we make this separately, when we need to modify docker command, we should modify two different lines. How about making a new function for this and call the same function at here and build function for maintainability?

File: os/tools/build_test.sh [hunk 3]
```diff
@@ -55,3 +55,12 @@ for target in ${build_targets[@]}; do
...
 done
+
+	echo "===================================="
+	echo "======== Clean build output ========"
+	echo "===================================="
+	cd ${OSDIR}; ${CLEAN_CMD}
+
+	echo "===================================="
+	echo "====== Build Test All Success ======"
+	echo "===================================="
```
Comment (sunghan-chang) [inline os/tools/build_test.sh:66]: 1. This is not a part of "adding build test option in dbuild". Let's split commits.
2.  The dbuild script prints start of build test, but this script does end. It looks strange.

File: os/dbuild.sh (hunk 2 above)
Comment (jeongarmy) [reply os/dbuild.sh:164]: will modify them with another PR.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**